pylint/checkers/strings.py

```diff
@@ -204,6 +204,12 @@
         "Used when we detect an f-string that does not use any interpolation variables, "
         "in which case it can be either a normal string or a bug in the code.",
     ),
+    "W1310": (
+        "Using formatting for a string that does not have any interpolated variables",
+        "format-string-without-interpolation",
+        "Used when we detect a string that does not have any interpolation variables, "
+        "in which case it can be either a normal string without formatting or a bug in the code.",
```

Comment:
We could use old_names to prevent legacy disable of ``too-many-format-args`` in user code to become useless-supression and for a new warning to appear instead. (I'm not sure if it's possible while keeping the old message in some other place, disregard this comment if it's causing problems.)

Reply:
But the messages are not really the same, right? Looking at other cases of `old_names` the new message often seems to exactly do the same, just separated in individual messages. Furthermore, looking at #4042 in some cases `too-many-format-args` is not currently raised. I think it might be better to really consider this a new checker, perhaps postponing it till 3.0?

Reply:
Some old names are not exactly the same for example we split up ``missing-docstrings`` in ``missing-module-docstring``, ``missing-class-docstrings``, ``missing-function-docstring``. The idea is to be able to change a name without forcing the user to change its disable or configuration, not having an exact match. So I think it's ok, to use this for a partial rename to have more precise message.

Reply:
Trying to pass test as we speak.
Have you considered the failing test on `pylint/config/__init__.py` and what to do with it?

Reply:
Ah, `too-many-format-args` already has an old-name.
Looking at 
```python
"W9012": (
            "Missing return type documentation",
            "missing-return-type-doc",
            "Please document the type returned by this method.",
            # we can't use the same old_name for two different warnings
            # {'old_names': [('W9007', 'missing-returns-doc')]},
        ),
``` 
this is not possible?

Reply:
Since then we can but it was a lot of work. Actually it's my first merge request to split up missing-docstring into multiple warning. Good time. Took 2 years and 10 MR. Search for C0111, C01112, C0114 or C0115 😄. By the way we can  probably uncomment the line and remove the comment, but maybe it's a bit risqué.

Reply:
https://github.com/PyCQA/pylint/pull/2036